core/CronArchive.php

```diff
@@ -452,6 +452,13 @@ public function run()
      */
     public function end()
     {
+        /**
+         * This event is triggered after archiving.
+         *
+         * @param CronArchive $this
+         */
+        Piwik::postEvent('CoreArchive.run.finish', array($this));
```

Comment:
I just realized those events (`CoreArchive.run.start` & `CoreArchive.run.finish`) were not placed within the `run()` method which is what they were supposed to be. Sorry I didn't notice they were placed in `init()` and `end()` method. 

In this case we should rename `CoreArchive.run.start` into `CoreArchive.init.start` and `CoreArchive.run.finish` into `CoreArchive.end` I think? 

As we haven't released a new version yet this should be no problem.